clients/python/odh_rules.mk

```diff
@@ -8,15 +8,15 @@ undeploy-mr-odh:
 
 .PHONY: test-e2e-odh
 test-e2e-odh:
-	@echo "Ensuring all extras are installed..."
-	poetry install --all-extras
```

Comment:
as these changes affect `test-e2e-odh` it is fine by me, but we might want to take this also upstream later to minimize potential conflicts, agree?

Reply:
odh_rules.mk does not exist in upstream, we it is only applicable to mid and downstream. So I intentionally introduced these targets in a separate file, to avoid possible conflicts.